test/cmdlineTests/standard_model_checker_targets_assert_bmc/output.json

```diff
@@ -2,66 +2,10 @@
     "errors": [
         {
             "component": "general",
-            "errorCode": "9207",
-            "formattedMessage": "Warning: 'transfer' is deprecated and scheduled for removal in the next breaking version (0.9). Use 'call{value: <amount>}(\"\")' instead.
-  --> A:11:7:
-   |
-11 | \t\t\t\t\t\ta.transfer(x);
-   | \t\t\t\t\t\t^^^^^^^^^^
-
-",
-            "message": "'transfer' is deprecated and scheduled for removal in the next breaking version (0.9). Use 'call{value: <amount>}(\"\")' instead.",
-            "severity": "warning",
-            "sourceLocation": {
-                "end": 234,
-                "file": "A",
-                "start": 224
-            },
-            "type": "Warning"
-        },
-        {
-            "component": "general",
-            "errorCode": "4661",
-            "formattedMessage": "Warning: BMC: Assertion violation happens here.
-  --> A:12:7:
-   |
-12 | \t\t\t\t\t\tassert(x > 0);
-   | \t\t\t\t\t\t^^^^^^^^^^^^^
-Note: Counterexample:
-  a = 0
-  x = 0
-
-Note: Callstack:
-Note:
-
-",
-            "message": "BMC: Assertion violation happens here.",
-            "secondarySourceLocations": [
-                {
-                    "message": "Counterexample:
-  a = 0
-  x = 0
-"
-                },
-                {
-                    "message": "Callstack:"
-                },
-                {
-                    "message": ""
-                }
-            ],
-            "severity": "warning",
-            "sourceLocation": {
-                "end": 258,
-                "file": "A",
-                "start": 245
-            },
-            "type": "Warning"
-        }
-    ],
-    "sources": {
-        "A": {
-            "id": 0
+            "formattedMessage": "parse error at line 7, column 458: syntax error while parsing object - unexpected string literal; expected '}'",
+            "message": "parse error at line 7, column 458: syntax error while parsing object - unexpected string literal; expected '}'",
+            "severity": "error",
+            "type": "JSONError"
         }
```

Comment:
This is not the expected output here. I think you broke it by not escaping the quotes.